File: Python/5KYU/MovingZerosToTheEnd.py
def move_zeros(array):
    i = 0
    j = len(array) - 1
    while i < len(array):
        if array[i] == 0 and i < j:
            array.append(array.pop(i))
            j -= 1
        else:
            i += 1
            
    return array

File: Go/5KYU/MovingZerosToTheEnd.go
package main

func MoveZeros(arr []int) []int {
	var index int = 0
	var stop int = len(arr)

	for index < stop {
		if arr[index] == 0 {
			(arr) = append((arr)[:index], append((arr)[index+1:], []int{0}...)...)
			stop -= 1
		} else {
			index += 1
		}
	}

	return arr
}
